x-pack/plugins/enterprise_search/public/applications/app_search/components/document_creation/creation_mode_components/paste_json_text.scss

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+.pasteJsonTextArea {
```

Comment:
Should we be prefixing our class names with something like `ent`, etc to avoid conflicts with other global styles?

I haven't looked at their build pipeline, but this is effectively a global style, right? 

`entPasteJsonTextArea`, `asPasteJsonTextArea`, etc.

This applies to more than this file, this is a largely overarching things I'm wondering about.

Like the upgrade_assistant plugin seems to prefix most of their class names with `upg`: https://github.com/elastic/kibana/blob/master/x-pack/plugins/upgrade_assistant/public/application/components/tabs/checkup/deprecations/_deprecations.scss

Reply:
I could be wrong but I'm 99% sure we're OK, since Kibana loads CSS dynamically per-plugin. So for instance WS's stylesheets aren't being loaded into our plugin and our AS SCSS files don't load into WS's plugin.

If whatever reason another plugin imported this scss file then they would get it, but that's probably intentional if so.

Reply:
As to why upgrade assistant has prefixes - it could be the plugin predates when Kibana loaded styles per-plugin, and they just never bothered to change? 🤷

Reply:
I guess the only thing it could conflict with is global styles, then. I mean, most likely not an issue, unless you use a very generic class name.

Reply:
Ya I'm not super worried because our CSS naming is still fairly specific [to components], also EUI is making the transition to emotion/CSS-in-JS in 2021 so it'll be a brave new world at that point in any case